uwblueprint/community-fridge-kw
# feat: Create scheduling Select Date Time page

## Notion ticket link

[Scheduling Flow - Select Date and Time](https://www.notion.so/uwblueprintexecs/Scheduling-Flow-Date-and-Time-e7476afb17464ce18f537acaedb5fd33)



## Implementation description
* Add select date and time page that consists of the following components:

1. Calendar to select a date
2. Select daypart (Early morning, Morning, Afternoon, Evening, Night)
3. Select drop-off time
4. Select frequency
5. Enter proposed end date (only render if this is NOT one time donation"



## Steps to test
1. Select a date in the datepicker. Only the dates for this week and next week can be selected. Everything else should be disabled.
2. Select a time of day
3. This should render the time slots. Each time slot should have icons representing number of ppl booked that slot.
4. Select frequency
5. Enter a proposed end date.
6. Validations?
7. _**Go to next page or go back and come back to this page. You should still see all the correct selected options.**_

Preferably, on the next page, do a `console.log(formValues)` and make sure all the formValues are correct.

https://user-images.githubusercontent.com/37950626/144347321-8915c538-7df4-4bdc-926f-86007f7a4494.mov



## What should reviewers focus on?
* 

## Checklist
